src/main/java/edu/cornell/kfs/rass/batch/AwardTranslationDefinition.java

```diff
@@ -84,34 +87,41 @@ public void processCustomTranslationForBusinessObjectCreate(
     }
 
     protected AwardAccount createDefaultAwardAccount(RassXmlAwardEntry xmlAward) {
-        Pair<String, String> defaultChartAndAccount = getChartAndAccountForDefaultAwardAccount();
-        String directorPrincipalId = parameterService.getParameterValueAsString(
-                RassStep.class, RassParameterConstants.DEFAULT_PROJECT_DIRECTOR);
+        Map<String, String> defaultAwardAccountAttributes = getDefaultAwardAccountAttributes();
+        String directorPrincipalId = parameterService.getParameterValueAsString(RassStep.class, RassParameterConstants.DEFAULT_PROJECT_DIRECTOR);
         if (StringUtils.isBlank(directorPrincipalId)) {
             throw new RuntimeException("Default project director parameter cannot be blank");
         }
-        
         AwardAccount awardAccount = new AwardAccount();
         awardAccount.setProposalNumber(xmlAward.getProposalNumber());
         awardAccount.setPrincipalId(directorPrincipalId);
-        awardAccount.setChartOfAccountsCode(defaultChartAndAccount.getLeft());
-        awardAccount.setAccountNumber(defaultChartAndAccount.getRight());
+        awardAccount.setChartOfAccountsCode(defaultAwardAccountAttributes.get(KFSPropertyConstants.CHART));
+        awardAccount.setAccountNumber(defaultAwardAccountAttributes.get(KFSPropertyConstants.ACCOUNT_NUMBER));
+        awardAccount.setActive(defaultAccountIndicatorIsActive(KFSPropertyConstants.ACCOUNT_ACTIVE_INDICATOR));
```

Comment:
This call to the indicator-is-active method is passing in the property key itself. I think it should instead be passing in the value that is associated with the key, given that the method treats its argument as being the value and not the key.